platform/android/export/export.cpp

```diff
@@ -1537,7 +1537,7 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 		args.push_back("-a");
 		args.push_back("android.intent.action.MAIN");
 		args.push_back("-n");
-		args.push_back(get_package_name(package_name) + "/com.godot.game.GodotApp");
+		args.push_back(get_package_name(package_name) + "/org.godotengine.godot.Godot");
```

Comment:
@creikey That's not correct. The new activity name for the Android template is `com.godot.game.GodotApp`. What you are changing back to is no longer valid.